AmericanMedicalAssociation/ama-style-guide-2
# [EWL-4611] SG2 | Create Agenda Molecule

## Ticket(s)

**Jira Ticket**
- [EWL-4609: SG2 | Create Agenda Date Atom](https://issues.ama-assn.org/browse/EWL-4609)
- [EWL-4611: SG2 | Create Agenda Molecule](https://issues.ama-assn.org/browse/EWL-4611)

## Description
As a user I should see an "Agenda" molecule with an h2 "Agenda Name", agenda date tab titles, and corresponding "Agenda Table" tab content.

This update creates the twig template, descriptive markdown, and relevant scss required to make the agenda tab and content along with the backstop test case and reference images. This PR also includes work for creating the agenda dates which cannot exist on their own.

Tab functionality leverages the jQuery UI tab library.

## To Test

- Pull `feature/EWL-4611-create-agenda-molecule` branch
- Run `gulp serve` in ama-style-guide-2/styleguide to ensure a local version of style guide is running.
- Navigate to [Molecule > Agenda](http://localhost:3000/?p=agenda-molecule).
- Confirm molecule looks as shown in Zeplin mock up.
- Click on the gear icon in the top right corner of the screen and then click "show pattern info". relevant information about the agenda molecule should appear.

## Visual Regressions

Reference images for the agenda molecule backstop tests have been included in this PR.


## Relevant Screenshots/GIFs
![image](https://user-images.githubusercontent.com/4438120/40187979-808185a8-59be-11e8-8fc8-d136ecdc4282.png)



## Remaining Tasks
- [x] Update glossary to include agenda definition.


## Additional Notes
N/A

---

[Guidelines for Contribution](CONTRIBUTING.md)
[SG2 Standards](ama-style-guide-2/docs/standards.md)
